src/main/java/co/cask/gcp/speech/SpeechToTextTransform.java

```diff
@@ -157,46 +96,39 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
     Schema recordSchema = getSchema(schema);
     List<StructuredRecord> speechArray = new ArrayList<>();
     StructuredRecord.Builder builder = StructuredRecord.builder(recordSchema);
-    for (SpeechRecognitionResult result: results) {
+    for (SpeechRecognitionResult result : results) {
       List<SpeechRecognitionAlternative> alternatives = result.getAlternativesList();
-      for (SpeechRecognitionAlternative alternative: alternatives) {
+      for (SpeechRecognitionAlternative alternative : alternatives) {
         float confidence = alternative.getConfidence();
         String transcript = alternative.getTranscript();
         StructuredRecord.Builder speech =
-          StructuredRecord.builder(schema.getField("speeches").getSchema().getNonNullable().getComponentSchema());
+          StructuredRecord.builder(recordSchema.getField("speeches").getSchema().getNonNullable().getComponentSchema());
         speech.set("confidence", confidence);
         speech.set("transcript", transcript);
         speechArray.add(speech.build());
       }
     }
     builder.set("speeches", speechArray);
```

Comment:
this should not be hardcoded, it should be a config property that can default to 'speeches'